docs/2020/index.md

```diff
@@ -1,15 +1,15 @@
 # OSG All-Hands Meeting 2020
 
-The [Open Science Grid](https://www.opensciencegrid.org) (OSG) and the
-[University of Oklahoma](https://www.ou.edu/) invite you to attend:
+The [Open Science Grid](https://www.opensciencegrid.org) (OSG)
+invites you to attend:
 
 <div style="border: 1px solid #3F51B5; color: #20295A; background-color: #E7E9F6; padding: 1ex; font-size: 115%; font-weight: bold;">
   31 August&nbsp;&ndash;&nbsp;3 September 2020
   <br>
-  The University of Oklahoma, Norman, OK
+  Virtual All-Hands Meeting
 </div>
 
-The detailed agenda will be posted later, check back here.
+The detailed agenda will be posted in July, check back here.
```

Comment:
The *detailed agenda* may not be ready in July....... Just the block schedule (overview). I think “later” is fine right now.